### ISSUE TITLE
Custom Web Searches Plugin entries disappear WOX 1.3.524

### ISSUE THREAD

**Issue by arunasdulskis** [Wox-launcher/Wox#2152]
### Are you submitting a bug report?

YES

After I create a custom search under Web Searches plugin they disappear from the list after few days. They don't show up with applying Action Keyword anymore, but if I try to create another search with the same keyword it says that the keyword is already in use.

Is it super annoying, as that's the main reason I need Wox for.

After creating the custom searches I don't mess with Wox settings anymore, regardless the entries disappear.

Please help!

Windows 10 Pro 64-bit
i7-8700
32GB Ram

**Comment by avxkim** [comment]
Devs are dead probably 🤣

**Comment by roose** [comment]
@arunasdulskis may be Wox unexpectedly closed(as I understand it saves that settings on exit) please go to `C:\Users\*youusername*\AppData\Roaming\Wox\Settings\Plugins\Wox.Plugin.WebSearch` and check at the end of `Settings.json` your custom web search, as example:
```
,
    {
      "Title": "kinopoisk",
      "ActionKeyword": "kp",
      "Icon": "favicon-152.png",
      "Url": "https://www.kinopoisk.ru/index.php?kp_query={q}",
      "Enabled": true
    }
```
If one of these fields is empty just delete this lines(backup file before)

If that file don't have you custom search then go `C:\Users\*youusername*\AppData\Roaming\Wox\Settings` search you custom search keyword in that `ActionKeywords` list and delete it and create new custom web search, after creating type `restart wox` in the Wox.
```
      "565B73353DBF4806919830B9202EE3BF": {
        "ID": "565B73353DBF4806919830B9202EE3BF",
        "Name": "Web Searches",
        "ActionKeywords": [
          "g",
          "wiki",
          "findicon",
          "facebook",
          "twitter",
          "maps",
          "translate",
          "duckduckgo",
          "github",
          "gist",
          "gmail",
          "drive",
          "wolframalpha",
          "stackoverflow",
          "lucky",
          "image",
          "youtube",
          "bing",
          "yahoo",
          "bd",
          "kp"
        ],
        "Disabled": false
      },
```

_sorry for my english_

**Comment by arunasdulskis** [comment]
@roose thank you, I was able to restore my custom searches.

Looks like WOX looses custom entries from C:\Users\*youusername*\AppData\Roaming\Wox\Settings\Plugins\Wox.Plugin.WebSearch
but maintains the entries in C:\Users\*youusername*\AppData\Roaming\Wox\Settings

I removed the keywords from the second file and was able to create the new searches.
Now I will test for how long those entries will remain there. Just in case they disappear again, I have made copies of both files, so hopefully next time it will be an easy fix.

Hope devs are reading this and will fix this issue
